# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/gtest/MockCall.h
# Commit: 0fc8c65e163e
# Full Hash: 0fc8c65e163ed5ccfc09c55ec906973cf35a79b4
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2018-11-03 09:44:53
# Regressor Bug: 1376873
# File Overlap Count: 3
# Description:
#   Bug 1376873 - Use audio/video sync groups; r=bwc
#   
#   This uses sync groups in the receive stream configs for the conduits rather
#   than establishing sync through direct calls. When the streams are created
#   in call.cc, ConfigureSync is called, which results in SetSync being called
# ==============================================================================

diff -r de3f5bd6846b -r 0fc8c65e163e media/webrtc/signaling/gtest/MockCall.h
--- a/media/webrtc/signaling/gtest/MockCall.h	Wed Feb 07 15:00:17 2018 -0500
+++ b/media/webrtc/signaling/gtest/MockCall.h	Fri Apr 27 07:25:19 2018 -0400
@@ -105,8 +105,6 @@
     return false;
   }
 
-  void SetSyncChannel(VoiceEngine* voice_engine, int audio_channel_id) override {}
-
   void EnableEncodedFrameRecording(rtc::PlatformFile file,
                                    size_t byte_limit) override {}
 